src/js/05-analytics.js

```diff
@@ -4,4 +4,6 @@
   if (!window.analytics) return
   const trackGithub = () => window.analytics.track('Clicked GitHub Link', { url: window.location.href })
   document.querySelectorAll('.github').forEach((githubLink) => githubLink.addEventListener('click', trackGithub))
+  const trackRSS = () => window.analytics.track('Clicked RSS Link')
+  document.querySelectorAll('.release-notes-rss').forEach((rssLink) => rssLink.addEventListener('click', trackRSS))
```

Comment:
can confirm that this gets sent to Segment, but only after I have removed the `not preview` condition seen here: https://github.com/mulesoft/docs-site-ui/blob/main/src/partials/head/head-scripts.hbs#L1C37-L1C50. Maybe we can improve this, like removing the `not preview` condition but label it instead, then in Segment, try to filter the labels to prevent them from being sent to the destinations.

![Uploading Screenshot 2024-02-01 at 11.34.20 AM.png…]()

Reply:
Excellent. Thank you for checking. What I'm hearing is that the data will go through OK on the live site. I could go back and forth on including the segment script on the preview site. It's certainly convenient for testing, but I wouldn't want to pollute the data either.

I think I would want to add a `test` field or something similar to our events when they're submitted from the preview site. I'll punt on this for now, but add a note to the spike if that's OK with you.

Reply:
yes, a spike would be great. Don't want to block you here